internal/bminventory/inventory.go

```diff
@@ -140,7 +140,7 @@ type InstallerInternals interface {
 	V2UpdateHostIgnitionInternal(ctx context.Context, params installer.V2UpdateHostIgnitionParams) (*models.Host, error)
 	GetCredentialsInternal(ctx context.Context, params installer.GetCredentialsParams) (*models.Credentials, error)
 	DownloadClusterFilesInternal(ctx context.Context, params installer.DownloadClusterFilesParams) (io.ReadCloser, int64, error)
-	V2DownloadClusterFilesInternal(ctx context.Context, params installer.V2DownloadClusterFilesParams) (io.ReadCloser, int64, error)
+	V2DownloadClusterFilesInternal(ctx context.Context, fileName, clusterId string) (io.ReadCloser, int64, error)
```

Comment:
Installer API is used by the controllers, and it needs to user the installer params as input arguments . In this case you can define an inner function that implements V2DownloadClusterFilesInternal functionality, and then make V2DownloadClusterFilesInternal and V2DownloadClusterCredentislaInternal call it